osquery/tables/system/darwin/signature.mm

```diff
@@ -91,92 +91,95 @@ void genSignatureForFile(const std::string& path, QueryData& results) {
   }
 
   // Actually validate.
+  bool isValidated = false;
   result =
       SecStaticCodeCheckValidityWithErrors(staticCode, flags, nullptr, nullptr);
   if (result == errSecSuccess) {
-    CFDictionaryRef codeInfo = nullptr;
-
-    result = SecCodeCopySigningInformation(
-        staticCode, kSecCSSigningInformation | kSecCSRequirementInformation,
-        &codeInfo);
-    if (result == errSecSuccess) {
-      // If we don't get an identifier for this file, then it's not signed.
-      CFStringRef ident =
-          (CFStringRef)CFDictionaryGetValue(codeInfo, kSecCodeInfoIdentifier);
-      if (ident != nullptr) {
-        // We have an identifier - this indicates that the file is signed,
-        // and, since it didn't error above, it's *also* a valid signature.
+    isValidated = true;
+  } else {
+    // If this errors, then we either don't have a signature, or it's malformed.
+    VLOG(1) << "Static code validity check failed for file: " << path;
```

Comment:
Before we bail when the signature is not validated, but an invalid signature still has useful info. There could be an app that is signed but not validated.